### DISCUSSION TITLE
Does it make sense to run Scenic+ on subset of cells instead of the whole population?

### DISCUSSION THREAD

**Discussion by alexwang1001** [aertslab/scenicplus#79] [General]
Hi Scenicplus team,
Thank you for providing this wonderful tool. I wonder, in your opinion, if it makes sense to run scenic plus on a subset of cells to identify some more detailed GRN differences between subtypes in a particular class of cells, say excitatory neurons.
Best,
Li

**Comment by SeppeDeWinter** [comment]
Hi @alexwang1001 

Yes this can make sense to do in some cases. However, you should make sure that you have some "background" cells (i.e. it will not work if the cells in the analysis are too homogeneous).

Best,

Seppe
